packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -15,6 +15,7 @@ export type AnnualHighlightCardsProps = {
 	};
 	titleHref?: string | null;
 	year?: string | number | null;
+	navigation?: object | null;
```

Comment:
I guess the change belongs to https://github.com/Automattic/wp-calypso/pull/71343 ?

Reply:
Good catch. I'll pull it out.